examples/profile-client/src/main/java/com/palantir/example/profile/ProfileStore.java

```diff
@@ -49,22 +50,22 @@ public ProfileStore(TransactionManager txnMgr, Transaction t) {
         this.t = t;
     }
 
-    private long getNewId() {
+    private UUID getNewId() {
         long result = -1;
         while (result < 0) {
             result = new Random().nextLong();
         }
-        return result;
+        return new UUID(new Random().nextLong(), new Random().nextLong());
```

Comment:
See also "all your entropy are belong to us".

On Thu, Feb 25, 2016 at 5:20 PM, Mark Elliot <notifications@github.com>
wrote:

> In
> examples/profile-client/src/main/java/com/palantir/example/profile/ProfileStore.java
> <https://github.com/palantir/atlasdb/pull/90#discussion_r54193143>:
>
> >          long result = -1;
> >          while (result < 0) {
> >              result = new Random().nextLong();
> >          }
> > -        return result;
> > +        return new UUID(new Random().nextLong(), new Random().nextLong());
>
> see also UUID.randomUUID() :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/palantir/atlasdb/pull/90/files#r54193143>.
>